plugins/logitech-scribe/meson.build

```diff
@@ -0,0 +1,17 @@
+if gusb.found()
```

Comment:
I think this needs some extra logic to only run on Linux.

Reply:
Added. Is there a dedicated host_machine for ChromeOS! Or it falls under linux umbrella!